store-frontend-broken-instrumented/store-frontend/app/views/spree/layouts/spree_application.html.erb

```diff
@@ -10,10 +10,25 @@
   type="text/javascript">
 </script>
 <script>
-  window.DD_RUM && window.DD_RUM.init({
+if (window.DD_RUM) {
+  window.DD_RUM.init({
     clientToken: '<%= ENV['DD_CLIENT_TOKEN'] %>',
     applicationId: '<%= ENV['DD_APPLICATION_ID'] %>',
+    sampleRate: 100,
+    trackInteractions: true,
+    service: 'storedog-ui',
+    env: 'production',
+    version: 1
```

Comment:
Can we switch this version to 1.1? Just to have some version comparison available.